software/base/src/main/java/brooklyn/entity/brooklynnode/BrooklynNodeImpl.java

```diff
@@ -202,6 +229,15 @@ protected void postStop() {
         }
     }
 
+    private boolean isStopMachine() {
```

Comment:
I was thrown by this method name when I looked at the impl.

Should this method be called `isStoppingMachine()` or `isMachineStopped()` (I prefer the former)?